src/edu/washington/escience/myria/PostgresBinaryTupleWriter.java

```diff
@@ -19,8 +19,11 @@
  */
 public class PostgresBinaryTupleWriter implements TupleWriter {
```

Comment:
Why not make this the `TupleWriter` implementation for persisting Postgres tables to HDFS? It should be much faster to re-ingest them into Postgres using `COPY`, right?